src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -26,6 +32,9 @@
 	@Option( names = "--help", usageHelp = true, hidden = true )
 	private boolean displayHelp;
 
+	@Option( names = "--exclude", description = "Ignore matching elements during accept." )
+	private final List<String> exclude = new ArrayList<>();
```

Comment:
Whenever not using an empty list, please use `Collections.emptyList()`
```suggestion
	private final List<String> exclude = Collections.emptyList();
```